builtin/providers/aws/resource_aws_vpc.go

```diff
@@ -5,9 +5,10 @@ import (
 	"log"
 	"time"
 
+	awsGo "github.com/awslabs/aws-sdk-go/aws"
```

Comment:
this should be OK without the namespace of `awsGo`. Any `awsGo.String()` or similar invocations can just be `aws.String()`